test/adjoint/test_fp_iteration.py

```diff
@@ -108,7 +117,16 @@ def test_no_late_convergence(self):
     def test_dropout(self, drop_out_converged):
         self.parameters.drop_out_converged = drop_out_converged
         mesh_seq = self.mesh_seq(time_partition=TimePartition(1.0, 2, [0.5, 0.5], []))
-        mesh_seq.fixed_point_iteration(oscillating_adaptor1, parameters=self.parameters)
+        solver_kwargs = (
+            {"track_coefficients": False}
+            if type(mesh_seq) is GoalOrientedMeshSeq
+            else {}
+        )
```

Comment:
Similarly here.

Reply:
Removed this entirely in https://github.com/mesh-adaptation/goalie/pull/259/commits/1ce741b79639c1b34bf104202e5693586fc168ae